### DISCUSSION TITLE
Cannot find cocotb-config

### DISCUSSION THREAD

**Discussion by ricardossilvestre** [cocotb/cocotb#3301] [Q&A] [answered]
I installed cocotb as user but cannot find the cocotb-config file.

The docs say:

> "After installation, you should be able to execute cocotb-config. If it is not found, you need to append its location to the PATH environment variable. This may happen when you use the --user option to pip, in which case the location is documented [here](https://docs.python.org/3/install/index.html#inst-alt-install-user)."

But it doesn't say where or how to find the cocotb-config file and add to the path. I also could not find it's location in the appended link

> tr.dynamic: command 'cocotb-config' could not be found in tr.dynamic policy.
> Makefile:15: /Makefile.sim: No such file or directory
> make: *** No rule to make target `/Makefile.sim'.  Stop.

**Comment by marlonjames** [comment, marked as answer]
`cocotb-config` is a Python console script, so you need the directory for script files in your path. The link lists where that directory should be.

**Comment by damnsavage** [comment]
The link above which I guess has the answer to this question gives a 404 error.  I can't find the cocotb-config script...
In my case I was able to find it using this command : find $(python -m site --user-base) -name cocotb-config
It was here ~/.local/bin so you'll need to add this directory to you path
